docs/developer-docs/latest/setup-deployment-guides/installation/platformsh.md

```diff
@@ -1,50 +1,38 @@
 ---
 title: Install using Platform.sh - Strapi Developer Documentation
 description: Quickly deploy a Strapi application using the official Platform.sh Strapi template.
+sidebarDepth: 1
 ---
 
-# Installing using Platform.sh
+# Platform.sh One-Click
 
-[Platform.sh](https://platform.sh/) gives you an easy way to get started and deploy your Strapi application.
+The following documentation will guide you through the one-click creation of a new Strapi project hosted on [Platform.sh](https://platform.sh/).
 
-You can find the template [source code](https://github.com/platformsh-templates/strapi#customizations) on Platform.sh GitHub for more information.
+Platform.sh is a Platform-as-a-Service that allows the management of multiple websites and applications. In particular, it allows to quickly install and deploy a Strapi application.
```

Comment:
I the Digital Ocean one, you wrote `Infrastructure as a Service (IaaS)` and here there is not (PaaS) at the end and you used `-`.
I think that we have to follow the format of the Digital Ocean one.

Reply:
Thanks Jim!

While I agree on the use of dashes (`-`) I'm not sure that Platform.sh and DigitalOcean offer the same type of services.
To me, DO is really more an Infrastructure as a Service. You can see, with the [dedicated config document in this PR](https://github.com/strapi/documentation/pull/326/files#diff-779eb19a35197514115db3cb48f933ed2032529b9d874d525491f1df26d1385c) that there are some DigitalOcean-specific configuration possibilities that, I believe, are not possible in Platform or Render — and I would define these two as Platform as a Service.

However, I am definitely not an expert on the difference between [PaaS vs IaaS](https://www.bmc.com/blogs/saas-vs-paas-vs-iaas-whats-the-difference-and-how-to-choose/)… 🤔